<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Communication Settings</string>
  </property>
  <widget class="QPushButton" name="comm_open">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comm_test">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>communication test</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comm_portlist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comm_baudrate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comm_bytesz">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comm_parity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comm_stopbits">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="comm_status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
